src/Bicep.Core/Semantics/Namespaces/SystemNamespaceType.cs

```diff
@@ -1249,6 +1252,14 @@ _ when TypeHelper.TryGetArmPrimitiveType(seedType) is TypeSymbol armPrimitive =>
                     .WithRequiredParameter("message", LanguageConstants.String, "The error message to use.")
                     .WithReturnType(LanguageConstants.Never)
                     .Build();
+
+                yield return new FunctionOverloadBuilder("loadDirectoryFileInformation")
```

Comment:
Any chance we could rename this to `loadDirectoryFileInfo` for the sake of a shorter name?